src/cirrus/build.py

```diff
@@ -277,8 +145,11 @@ def main():
 
     Execute build command
     """
-    opts = build_parser(sys.argv)
-    execute_build(opts)
+    opts, extras = build_parser(sys.argv)
+    try:
+        plugin_build(opts, extras)
+    except Exception:
+        sys.exit(1)
```

Comment:
I'd suggest removing the try/catch here.  If the plugin throws an exception we'll still exit but at least we'll get to see the traceback.